themes/theme-ios11/pages/More/components/UserMenu/components/LoggedIn/index.jsx

```diff
@@ -14,40 +15,38 @@ import Item from '../../../Item';
 const LoggedIn = ({ logout }) => {
   const props = { Item };
   return (
-    <Fragment>
-      <Portal name={commonPortals.NAV_MENU_MY_ACCOUNT_BEFORE} props={props} />
-      <Portal name={commonPortals.NAV_MENU_MY_ACCOUNT} props={props}>
-        <div data-test-id="userMenu">
-          <Section title="navigation.your_account">
-            <Item
-              href={PROFILE_PATH}
-              label="navigation.profile"
-              testId="accountButton"
-            />
-            <Item
-              href={ORDERS_PATH}
-              label="navigation.order_history"
-              testId="accountButton"
-            />
-            <Item
-              href={WISH_LIST_PATH}
-              label="navigation.favorites"
-              testId="accountButton"
-            />
-            <Portal name={commonPortals.NAV_MENU_LOGOUT_BEFORE} props={props} />
-            <Portal name={commonPortals.NAV_MENU_LOGOUT} props={props}>
+    <SurroundPortals portalName={NAV_MENU_MY_ACCOUNT} portalProps={props}>
+      <div data-test-id="userMenu">
+        <Section title="navigation.your_account">
+          {hasNewServices() && (
+            <>
               <Item
-                onClick={logout}
-                label="navigation.logout"
-                testId="logoutButton"
+                href={PROFILE_PATH}
+                label="navigation.profile"
+                testId="accountButton"
               />
-            </Portal>
-            <Portal name={commonPortals.NAV_MENU_LOGOUT_AFTER} props={props} />
-          </Section>
-        </div>
-      </Portal>
-      <Portal name={commonPortals.NAV_MENU_MY_ACCOUNT_AFTER} props={props} />
```

Comment:
if I see correctly, you removed the portals NAV_MENU_MY_ACCOUNT_AFTER and NAV_MENU_LOGOUT_AFTER as well as their BEFORE parts. But aren't they used by some extensions maybe?

Reply:
if you check what the `SurroundPortals` component does, you'll see that it renders the before and after portals. usage of this component reduces boilerplate code and an this case it also allowed to remove the `Fragment`